$MirOS$

	Upstream (Lynx) should at least enable KANJI_CODE_OVERRIDE by
	default and incorporate the #ifndef LYNX_VERSION part.
	Setting PREFERRED_LANGUAGE to "" is a sensible thing to do too.

--- userdefs.h.orig	Thu May 17 22:52:59 2007
+++ userdefs.h	Sun May 20 11:17:21 2007
@@ -333,7 +333,7 @@
  * use any default viewers for image types.  Note that open is used as
  * the default for NeXT, instead of the XLOADIMAGE_COMMAND definition.
  */
-#define XLOADIMAGE_COMMAND "xli %s &"
+#define XLOADIMAGE_COMMAND "xloadimage -quiet %s"
 
 /**************************
  * For UNIX systems, SYSTEM_MAIL and SYSTEM_MAIL_FLAGS are set by the
@@ -461,7 +461,7 @@
  * Normally we expect you will connect to a remote site, e.g., the Lynx starting
  * site:
  */
-#define STARTFILE "http://lynx.isc.org/"
+/* #define STARTFILE "http://lynx.isc.org/" */
 /*
  * As an alternative, you may want to use a local URL.  A good choice for this
  * is the user's home directory:
@@ -471,6 +471,7 @@
  * you can connect to reliably.  Otherwise users will become confused and think
  * that they cannot run Lynx.
  */
+#define STARTFILE "file://localhost/~/"
 
 /*****************************
  * HELPFILE must be defined as a URL and must have a
@@ -727,7 +728,7 @@
  * that preference in their RC file.  This may be a comma-separated list
  * of languages in decreasing preference.
  */
-#define PREFERRED_LANGUAGE "en"
+#define PREFERRED_LANGUAGE ""
 
 /*****************************
  * PREFERRED_CHARSET specifies the character set in MIME notation (e.g.,
@@ -802,8 +803,8 @@
  * and use "http://" as the default (e.g., gopher.wfbr.edu or gopher.wfbr.
  * will be made gopher://gopher.wfbr.edu).
  */
-#define URL_DOMAIN_PREFIXES "www."
-#define URL_DOMAIN_SUFFIXES ".com,.edu,.net,.org"
+#define URL_DOMAIN_PREFIXES ""
+#define URL_DOMAIN_SUFFIXES ""
 
 /********************************
  * If LIST_NEWS_NUMBERS is set TRUE, Lynx will use an ordered list
@@ -844,7 +845,7 @@
  * directory.  If it is in a subdirectory, begin it with a dot-slash
  * (e.g., ./lynx/.lynxsig).  The definition here can be changed in lynx.cfg.
  */
-#define LYNX_SIG_FILE ".lynxsig"
+#define LYNX_SIG_FILE ".etc/sig"
 
 /********************************
  * BIBP_GLOBAL_SERVER is the default global server for bibp: links, used
@@ -1371,7 +1372,9 @@
  * the version definition with the Project Version on checkout.  Just
  * ignore it. - kw */
 /* $Format: "#define LYNX_VERSION \"$ProjectVersion$\""$ */
+#ifndef LYNX_VERSION
 #define LYNX_VERSION "2.8.7dev.5"
+#endif
 #define LYNX_WWW_HOME "http://lynx.isc.org/"
 #define LYNX_WWW_DIST "http://lynx.isc.org/current/"
 /* $Format: "#define LYNX_DATE \"$ProjectDate$\""$ */
@@ -1559,11 +1562,11 @@
 
 /*****************************
  * Uncomment the following line to enable the kanji code override routine.
- * The code can be changed by pressing ^L.  More precisely, this allows
+ * The code can be changed by pressing Shift-J.  More precisely, this allows
  * the user to override the assumption about the kanji code for the document
  * which Lynx has made on the basis of a META tag and HTTP response.
  */
-/*#define KANJI_CODE_OVERRIDE */
+#define KANJI_CODE_OVERRIDE
 
 
 /****************************************************************
